src/app/waste-water/waste-water-analysis/waste-water-analysis.service.ts

```diff
@@ -59,14 +59,16 @@ export class WasteWaterAnalysisService {
   initXAxisVariables() {
     let analysisXAxisVariables: Array<DataTableVariable> = JSON.parse(JSON.stringify(DataTableVariables));
     analysisXAxisVariables.forEach(x => x.selected = false);
+    // Shift SRT and MLSS to start
+    analysisXAxisVariables.some((xVar, i, variables) => xVar.name == 'MLSS' && variables.unshift(variables.splice(i,1)[0]));
```

Comment:
Can you just update DataTableVariables list?

Reply:
We don't want SRT to be selectable but I did move up MLSS and improved a bit.